developer-docs-site/docs/nodes/staking.md

```diff
@@ -0,0 +1,236 @@
+
+When you satisfy the above two minimum requirements, then you can join the validator set at any time, start validating and earn rewards.
+
+:::tip Joining the validator set
+For step-by-step instructions on how to join the validator set, see: [Joining Validator Set](https://aptos.dev/nodes/ait/connect-to-testnet#joining-validator-set).
+:::
+
+### Minimum and maximum stake
+
+You must stake the required minimum amount to join the validator set. Moreover, you can only stake up to the maximum stake amount. 
+
+If at any time after joining the validator set, your current staked amount exceeds the maximum allowed stake (for example as the rewards are added to your staked amount), then your voting power and the rewards will be calculated only using the maximum allowed stake amount, and not your current staked amount. 
+
+### When the staked amount falls below minimum
+
+If after joining the validator set, at the start of an epoch your stake drops below the minimum required amount, then you will be removed from the validator set. 
+
+:::tip
+In the current version of the staking on the Aptos blockchain, there is no possibility of your stake dropping below the required minimum before the lockup period expires. **This will change when Aptos implements slashing, i.e., penalty for malicious validator behavior**.
+:::
+
+### Minimum and maximum lockup period
+
+To join the validator set, you must lockup your stake for the required minimum lockup period (in seconds). You cannot withdraw any staked amount before the expiry of the lockup period. 
+
+Moreover, you can only lockup your stake for the maximum allowed period (in seconds).
+
+However, if your remaining lockup time falls below the required minimum lockup period, then you will **not** be removed from the validator set. 
+
+:::tip Note the difference
+If your staked amount falls below the required minimum you will be removed from the validator set, but if your remaining lockup time falls below the required minimum lockup period, you will not be removed from the validator set.
+:::
+
+### When the lockup period expires
+
+When your lockup period expires, you can either extend the lockup period so you can continue to validate and receive the rewards, or you can withdraw your total staked amount and stop validating.
+
+:::tip Set by the governance
+
+The above minimum and maximums for staked amount and the lockup period are decided by the Aptos governance and not by any special entity like the Aptos Labs. These minimum and maximum configurations are controlled by the covenants that the Aptos community members vote on.
+:::
+
+## Epoch
+
+An epoch in the Aptos blockchain is defined as a duration of time, in seconds, during which a number of blocks are voted on by the validators, the validator set is updated, and the rewards are distributed to the validators. 
+
+:::tip
+Currently an epoch on the Aptos blockchain is defined as sixty seconds (one hour).
```

Comment:
sixty minutes?

Reply:
3600 seconds? (since it says "is defined as a duration of time, in seconds")